core/network/fabric-interop-cc/contracts/interop/manage_assets.go

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// manage_assets is a chaincode that contains all the code related to asset management operations (e.g., Lock, Unlock, Claim)
+// and any related utility functions
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"crypto/sha256"
+	"encoding/base64"
+	"time"
+
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+	"github.com/golang/protobuf/proto"
+	log "github.com/sirupsen/logrus"
+	"github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/contracts/interop/protos-go/common"
+)
+
+type AssetLockValue struct {
+	Locker	string	`json:"locker"`
+	Recipient	string	`json:"recipient"`
+	Hash	string	`json:"hash"`
```

Comment:
Perhaps align the type declarations vertically using tabs and whitespaces?

Reply:
Done @VRamakrishna